include/shell/shell.h

```diff
@@ -97,7 +114,32 @@ struct shell_static_entry {
  * @param[in] help    Pointer to a command help string.
  * @param[in] handler Pointer to a function handler.
  */
-#define SHELL_CMD_REGISTER(syntax, subcmd, help, handler)		   \
+#define SHELL_CMD_ARG_REGISTER(syntax, subcmd, help, handler,		   \
```

Comment:
I would update a bit macro description to exaplain difference between `SHELL_CMD_REGISTER` and `SHELL_CMD_ARG_REGISTER`

Reply:
Ok